src/pke/lib/schemebase/base-pke.cpp

```diff
@@ -170,14 +170,15 @@ std::shared_ptr<std::vector<Element>> PKEBase<Element>::EncryptZeroCore(
 // makeSparse is not used by this scheme
 template <class Element>
 std::shared_ptr<std::vector<Element>> PKEBase<Element>::EncryptZeroCore(
-    const PublicKey<Element> publicKey,
-    const std::shared_ptr<ParmType> params) const {
+    const PublicKey<Element> publicKey, 
+    const std::shared_ptr<ParmType> params,
+    const DggType &dgg) const {
   const auto cryptoParams =
       std::static_pointer_cast<CryptoParametersRLWE<Element>>(
           publicKey->GetCryptoParameters());
 
   const auto ns = cryptoParams->GetNoiseScale();
-  const DggType &dgg = cryptoParams->GetDiscreteGaussianGenerator();
+  //const DggType &dgg = cryptoParams->GetDiscreteGaussianGenerator();
```

Comment:
remove this commented line

Reply:
fixed

Reply:
don't mark it fixed unless it is in the branch